scripts/travis_ci/series_ci.sh

```diff
@@ -37,4 +37,4 @@ curl \
   --header "Authorization: Token 27d62d1f-71c6-4075-95b5-f7aec56d0204" \
   --header "Content-Type: application/json" \
   --data "{\"values\":[{\"value\":\"$ram\",\"line\":\"RAM\"},{\"value\":\"$flash\",\"line\":\"FLASH\"}],\"sha\":\"$SHA\"}" \
-  https://seriesci.com/api/UBCFormulaElectric/Consolidated-Firmware/$SERIES_NAME/many
+  -k https://seriesci.com/api/UBCFormulaElectric/Consolidated-Firmware/$SERIES_NAME/many
```

Comment:
I was getting this error message while building in travis ci: 
`curl: (60) SSL certificate problem: certificate has expired .sh`

[This](https://stackoverflow.com/questions/18964175/how-to-fix-curl-60-ssl-certificate-invalid-certificate-chain) seemed to fix it

Reply:
LGTM lets get this 🍞

Reply:
* Lets get this 🍆

Reply:
PLEASE APPROVE PLEASE APPROVE

Reply:
Added a comment actually

Reply:
NIT NIT